lib/primo/search.rb

```diff
@@ -51,7 +51,18 @@ def self.get(params = {})
       params = params.to_h.transform_keys(&:to_sym)
       method = get_method params
       (url, query) = url(params)
-      new super(url, query:, timeout: Primo.configuration.timeout(params)), method
+
+      retry_count = Primo.configuration.enable_retries ? Primo.configuration.retries : 0
+      begin
+        new super(url, query:, timeout: Primo.configuration.timeout(params)), method
+      rescue Net::ReadTimeout
+        if (Primo.configuration.retries && (retry_count -= 1) > 0)
```

Comment:
This is hard to process here and seems redundant with line 55:

If you are worried that Primo.configuration.retries is nil you can set a default for it when setting retry_count'